test/controllers/teams_controller_test.rb

```diff
@@ -0,0 +1,57 @@
+require "test_helper"
+
+class TeamsControllerTest < ActionController::TestCase
+
+  setup do
+    @team = teams(:one)
+    sign_in users(:nancy)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:teams)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  def test_should_get_settings
+    get :settings, id: @team
+    assert_response :success
+  end
+
+  test "should create team" do
+    assert_difference('Team.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.